README.md

```diff
@@ -2264,6 +2265,46 @@ Checks if value is an integer.
 
 **[⬆ back to top](#quick-links)**
 
+### _.isPlainObject
+
+Checks if value is a plain object, that is, an object created by the Object constructor or one with a [[Prototype]] of null.
+
+  ```js
+  var object = { 'a': 1, 'b': 2, 'c': 1 };
+
+  // Underscore/Lodash
+  var result = _.isPlainObject(object);
+  console.log(result)
+  // output: true
+
+  function isPlainObject(value) {
+    if (typeof value !== 'object' || value === null) return false
+
+    if (Object.prototype.toString.call(value) !== '[object Object]') return false
+
+    const proto = Object.getPrototypeOf(value);
+    if (proto === null) return true
+
+    const Ctor = Object.prototype.hasOwnProperty.call(proto, 'constructor') && proto.constructor;
+    return (
+        typeof Ctor === 'function' &&
+        Ctor instanceof Ctor && Function.prototype.call(Ctor) === Function.prototype.call(value)
+    );
+
+    var result = invert(object);
+    console.log(result)
+    // output: true
+    }
```

Comment:
These lines are not part of the isPlainObject() function.

Reply:
Please send a PR to fix anything